docs/book/src/developer/architecture/controllers/control-plane.md

```diff
@@ -44,12 +44,19 @@ The `ImplementationControlPlane` *must* rely on the existence of
 
 ### CRD contracts
 
-#### Required `spec` fields for implementations using replicas
```

Comment:
why was `for implementations using replicas` dropped here? replicas should be an optional field for managed providers e.g. EKS, AKS

Reply:
same question w.r.t the type definitions having the field as required: https://github.com/kubernetes-sigs/cluster-api/commit/fb67f57c2c33aa9b16864cead48f8ec42966567a#diff-9fffa8bb7ae45833301caf7cfa3f6e25c83f705e1369a35a6cb1c25dcc78bc97R98-R99

Reply:
(fyi) This was just discussed in the CAPI meeting and it will be changed/fixed

Reply:
thanks for the heads up!

Reply:
Sorry if I missed this point, hopefully it is fixed now.

Requirements for Cluster Class compatibility are in a separated section and they should not affect anymore the possibility of having control plane providers to opt out from supporting replicas